doc/source/development/building_from_source.rst

```diff
@@ -1,62 +1,74 @@
-.. _build_hints:
+.. include:: ../substitutions.rst
+
+.. _building_from_source:
```

Comment:
would you mind adding in doc/source/_extensions/redirects.py a redirect from /build_hints.html to /development/building_from_source.html  as I suspect there must a number of public traces to http://gdal.org/build_hints.html

Reply:
Thanks for the hint, I figured there must be some way to do this.